common/src/main/java/edu/snu/nemo/common/test/ArgBuilder.java

```diff
@@ -20,11 +20,14 @@
 import java.util.List;
 import java.util.StringJoiner;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 /**
  * Argument builder.
  */
 public final class ArgBuilder {
+  private static final List<List<String>> DEFAULT_ARGS = Arrays.asList(Arrays.asList("-executor_json",
```

Comment:
These changes may conflict with `master`. Please follow `master` to resolve this.